### PR TITLE
3.2.x: Correct the deb packages filenames in the install commands

### PR DESCRIPTION
**Pull request by lluiscampos**
Cherry-pick of #1676, #1677, #1683 and #1685.

### CODE REVIEW

File: 09.Downloads/docs.md [hunk 1]
```diff
@@ -382,27 +382,27 @@ download it by running:
...
 [/ui-tabs]
 
 
 Then install the package with:
 
-<!--AUTOVERSION: "mender-monitor_%-1_all.deb"/monitor-client -->
+<!--AUTOVERSION: "mender-monitor_%-1"/monitor-client -->
 ```bash
-dpkg -i mender-monitor_1.1.0-1_all.deb
+dpkg -i mender-monitor_1.1.0-1%2Bdebian%2Bbuster_all.deb
```
Comment (merlin-northern) [inline 09.Downloads/docs.md]: wait so we are now changing to URL encoded debs?

File: 09.Downloads/docs.md [hunk 2]
```diff
@@ -414,25 +414,25 @@ through the package:
...
 [/ui-tabs]
 
 
 Then install the package with:
 
-<!--AUTOVERSION: "mender-monitor_demo_%-1_all.deb"/monitor-client -->
+<!--AUTOVERSION: "mender-monitor_demo_%-1"/monitor-client -->
 ```bash
-dpkg -i mender-monitor_demo_1.1.0-1_all.deb
+dpkg -i mender-monitor_demo_1.1.0-1+debian+buster_all.deb
```
Comment (merlin-northern) [inline 09.Downloads/docs.md:437]: this package does not exist, the correct file name is:
```
mender-monitor-demo_1.1.0-1_all.deb
```

**Review by merlin-northern — COMMENTED** [review]
one comment.

File: 09.Downloads/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:437]: What? That is from the first cherry-pick, that was wrong, and then amended in the following commits. Please look at the final diff.

File: 09.Downloads/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md]: Yes, that was the original cherry-pick (which was wrong) and then it is amended.

File: 09.Downloads/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:437]: Cherry-picked now #1683.

File: 09.Downloads/docs.md [hunk 3]
```diff
@@ -382,27 +382,27 @@ download it by running:
...
 [/ui-tabs]
 
 
 Then install the package with:
 
-<!--AUTOVERSION: "mender-monitor_%-1_all.deb"/monitor-client -->
+<!--AUTOVERSION: "mender-monitor_%-1"/monitor-client -->
 ```bash
-dpkg -i mender-monitor_1.1.0-1_all.deb
+dpkg -i mender-monitor_1.1.0-1+debian+buster_all.deb
```
Comment (merlin-northern) [inline 09.Downloads/docs.md:405]: see this comment: https://github.com/mendersoftware/mender-docs/pull/1678/files#r808873850

Comment (lluiscampos) [reply 09.Downloads/docs.md:407]: Cherry-picked #1685 now.

**Pull request merged**